core/src/main/java/org/akaza/openclinica/dao/core/SchedulerResources.java

```diff
@@ -36,7 +36,7 @@ public void setSchemaInfo() {
         }
         if (StringUtils.isEmpty(schema))
             schema = CoreResources.getField("schema");
-        System.out.println("SchedulerResources**********Using schema:" + schema);
+        logger.debug("SchedulerResources**********Using schema: {}",schema);
```

Comment:
General comment: you should have a space after each argument. 
E.g. In this case, there should be a space after comma.

Reply:
I have changed according to your comments